app/controllers/setup/two.js

```diff
@@ -169,11 +169,15 @@ export default Controller.extend(ValidationEngine, {
     _oauthSetup() {
         let blogTitle = this.get('blogTitle');
         let config = this.get('config');
+        let promises = [];
+
+        promises.pushObject(this.get('settings').fetch());
+        promises.pushObject(this.get('config').fetchPrivate());
 
         this.get('hasValidated').addObjects(['blogTitle', 'session']);
 
         return this.validate().then(() => {
-            return this.get('settings').fetch()
+            return RSVP.all(promises)
```

Comment:
This isn't needed here, we still run `this._afterAuthentication` which loads settings & config - the reason the settings fetch is here is to make sure we have all the settings before saving the blog title rather than fetching everything once auth has completed.

Probably moot anyway as the oauth code will be removed shortly 🙈